cfgov/cfgov/settings/production.py

```diff
@@ -148,7 +148,7 @@
         root_url=SAML_ROOT_URL,
         acs_path=reverse_lazy("saml2_acs"),
     )
-    ACS_DEFAULT_REDIRECT_URL = reverse_lazy("check_permissions")
+    ACS_DEFAULT_REDIRECT_URL = reverse_lazy("wagtailadmin_home")
```

Comment:
I have no way to test this at present, but I believe this should be the correct post-SAML2-AUTH redirect with our `check_permissions` view removed.